rasa/nlu/training_data/formats/markdown.py

```diff
@@ -37,13 +37,15 @@
 AVAILABLE_SECTIONS = [INTENT, SYNONYM, REGEX, LOOKUP]
 MARKDOWN_SECTION_MARKERS = [f"## {s}:" for s in AVAILABLE_SECTIONS]
 
-item_regex = re.compile(r"\s*[-*+]\s*(.+)")
+item_regex = re.compile(r"\s*[-*+]\s*((?:.+\s*)*)")
```

Comment:
Why did you changed this?

Reply:
The `item_regex` will fail in case of having whitespace tokens in the line since they aren't escaped anymore.

```
import re

item_regex = re.compile(r"\s*[-*+]\s*((?:.+\s*)*)")
old_item_regex = re.compile(r"\s*[-*+]\s*(.*)")

t = '- THIS IS A MD ENTRY WITH a "\\n" token'
print(repr(t)) 
# '- THIS IS A MD ENTRY WITH a "\\n" token'

unespaced_t = decode_string(t)
print(repr(unespaced_t)) 
# '- THIS IS A MD ENTRY WITH a "\n" token'

re.match(old_item_regex, unespaced_t).groups(0)[0] 
#'THIS IS A MD ENTRY WITH a "'

re.match(item_regex, unespaced_t).groups(0)[0] 
# 'THIS IS A MD ENTRY WITH a "\n" token'

```